extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -5,15 +5,14 @@
  * A Wall consists of "Bricks" which are each a single topic/thread/conversation.
  * In typical use, a Wall will only load a subset of Bricks because there will be a TON of bricks as time goes on.
  */
+
 class WallBaseController extends WikiaController {
-	const WALL_MESSAGE_RELATIVE_TIMESTAMP = 604800; // relative message timestamp for 7 days (improvement 20178)
+	const WALL_MESSAGE_RELATIVE_TIMESTAMP = 604800; // relative message timestampt for 7 days (improvement 20178)
 	const DEFAULT_MESSAGES_PER_PAGE = 10; // how many messages should appear per page if not specified otherwise
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)